### PR TITLE
[WIP] chore: code analyzer unused checks

### PR DESCRIPTION
**Pull request by drminh2807**

### CODE REVIEW

File: lib/config/app_emojis.dart [hunk 1]
```diff
@@ -1,30 +0,0 @@
-abstract class AppEmojis {
-  static const List<String> emojis = [
```
Comment (hoangdat) [inline lib/config/app_emojis.dart:2]: we should keep it for message reaction?

File: pubspec.yaml [hunk 2]
```diff
@@ -38,8 +36,6 @@ dependencies:
     sdk: flutter
   flutter_map: ^3.1.0
   #  flutter_matrix_html: ^1.1.0
-  flutter_olm: ^1.2.0
-  flutter_openssl_crypto: ^0.1.0
```
Comment (hoangdat) [inline pubspec.yaml:42]: keep it

File: scripts/code_analyze.sh [hunk 3]
```diff
@@ -1,4 +1,5 @@
 #!/bin/sh -ve
 flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs
 dart format --set-exit-if-changed lib/ test/
-flutter analyze
+dart run dart_code_metrics:metrics check-unused-files lib
```
Comment (tk-nguyen) [inline scripts/code_analyze.sh:4]: Please note, `dart-code-metrics` is discontinued and now requires a license: https://github.com/dart-code-checker/dart-code-metrics.

Comment (tk-nguyen) [reply scripts/code_analyze.sh:4]: Not that I know of.

Comment (drminh2807) [reply scripts/code_analyze.sh:4]: dont know another lib
also this check cannot ignore files
so this PR is pending

**Pull request closed without merging**